ClosedXML_Tests/Excel/RichText/XLRichStringTests.cs

```diff
@@ -668,5 +669,43 @@ public void ToStringTest()
             actual = richString.ToString();
             Assert.AreEqual(expected, actual);
         }
+
+        [Test(Description = "See #1361")]
+        public void CanClearInlinedRichText()
+        {
+            using (var outputStream = new MemoryStream())
+            {
+                using (var inputStream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Other\InlinedRichText\inputfile.xlsx")))
+                using (var workbook = new XLWorkbook(inputStream))
+                {
+                    workbook.Worksheets.First().Cell("A1").Value = "";
+                    workbook.SaveAs(outputStream);
+                }
+
+                using (var wb = new XLWorkbook(outputStream))
+                {
+                    Assert.AreEqual("", wb.Worksheets.First().Cell("A1").Value);
+                }
+            }
+        }
+
+        [Test]
+        public void CanChangeInlinedRichText()
+        {
+            using (var outputStream = new MemoryStream())
+            {
+                using (var inputStream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Other\InlinedRichText\inputfile.xlsx")))
+                using (var workbook = new XLWorkbook(inputStream))
+                {
+                    workbook.Worksheets.First().Cell("A1").RichText.AddText("Changed");
```

Comment:
> I have to look at this in detail still, but I believe the actual solution was to add full support for inline rich text strings so that loading a file that contains it populates the IXLCell.RichText property. 

It's been loaded fine:
![image](https://user-images.githubusercontent.com/19576939/76500322-c9e02b80-6459-11ea-893f-1c146de8eeac.png)

And then saved too - but as a shared string.